result.go

```diff
@@ -72,3 +72,34 @@ func (r solution) Attempts() int {
 func (r solution) InputHash() []byte {
 	return r.hd
 }
+
+func stripVendor(path string, info os.FileInfo, err error) error {
+	if info.Name() == "vendor" {
+		if _, err := os.Lstat(path); err == nil {
+			symlink := (info.Mode() & os.ModeSymlink) != 0
+			dir := info.IsDir()
+
+			switch {
+			case symlink && dir:
+				// This must be a windows junction directory. Support for these in the
+				// standard library is spotty, and we could easily delete an important
+				// folder if we called os.Remove. Just skip these.
```

Comment:
nit: is it `os.Remove()` we'd call here, or `os.RemoveAll()`? honestly, i don't even know, b/c idk whether to think of those junction dirs as files or dirs 😛